docs/chapter_phil/What is probability? Bayes contra Hume.rst

```diff
@@ -1,3 +1,801 @@
+But a few names stand out in this history of probability – Bayes, Laplace, 
+Fisher, Neyman and Pearson - because they are basic to our understanding of 
+probability in statistics and its opposing school, so let´s zoom in on them.
+
+Thomas Bayes (1702-1761) was a minister and a member of the Royal Society with 
+only one publication under his name during his lifetime. He became the name 
+giver of the ‘Bayesian school’ or the subjective probability when his 
+friend Richard Price published a manuscript of his after his death. The 
+manuscript was on the problem of how one may obtain the probability of a 
+hypothesis given some data – something we would later call the Bayes’ 
+theorem. This 
+concept was as simple as it was beautiful – We modify our opinion with 
+objective information: Initial beliefs + recent objective data = new and 
+improved belief (`McGrayne, 2011`_). We will have a closer look at this theorem 
+later in this chapter.
+
+Bayes’ ideas were picked up by Pierre-Simon Laplace (1749-1827) in the early 
+19th century. Laplace was a probabilist and even referred to as the “Newton 
+of France” for his work in physics and astronomy. In his lifetime he came up 
+with two theories of probability that showed more or less the same results with 
+very large samples (`Galavotti, 2015`_). Just on a quick side note – Laplace 
+despised gambling.
+
+Times changed; politics as well as personal animosities played their part as 
+much as a shift in the idea what ‘science’ could do, and with it crashed 
+the idea that there is a balance between subjective beliefs and objective 
+frequencies. The new natural science wanted a ‘route to certain knowledge’, 
+and how could this be subjective in any way? This led to scientists treating 
+the two points of view in probability as diametrical (`McGrayne, 2011`_). 
+Soon mathematicians should take over the field of probability, bringing with 
+them the frequentist approach or objective probability and most terms, concepts 
+and techniques we use in statistics today (`Dienes, 2008`_).
+
+The most dominant name in this upcoming approach would become Sir Ronald Fisher 
+(1890-1962), a geneticist, who coined the terms ‘null hypothesis’ and 
+‘significance’, and suggested a p-value of .05 as an arbitrary level to judge a 
+result statistically significant (`Dienes, 2008`_). After working for years on 
+randomization, sampling theory, experimental designs, tests of significance and 
+analysis of variance he published a revolutionary manual for 
+non-statisticians – Statistical Methods for Research Workers (`Fisher, 
+1925`_). This manual was a game changer and helped to turn the frequentist 
+approach into the state of the art for years to come.
+
+Aside from (or maybe because of) his smart mind, Fisher was in a constant 
+ad-hominem feud with his colleagues, especially Karl Pearson (1857-1936) – a 
+fanatical eugenicist and name-giver of the Pearson correlation, his son Egon 
```

Comment:
It seems unfair to call out Pearson for being an eugenicist and not also call out Fisher for being an eugenicist too, an [opponent of the UN declaration of equality of the 'races'](https://en.wikipedia.org/wiki/Ronald_Fisher#Race), a science denier, ...

Reply:
I mean, either note it for both, or for neither.

Reply:
agreed, would remove it for Pearson.